vllm/_custom_ops.py

```diff
@@ -266,6 +270,17 @@ def awq_gemm(input: torch.Tensor, qweight: torch.Tensor, qzeros: torch.Tensor,
     return torch.ops._C.awq_gemm(input, qweight, qzeros, scales, split_k_iters)
 
 
+def awq_hpu_gemm(
```

Comment:
Shouldnt it be added to vllm-hpu-extension/ops ?

Reply:
I followed the convention of cuda implementation https://github.com/HabanaAI/vllm-fork/blob/f2fd3586394a3b08fc5d26164a702d0fd4e1d6e7/vllm/_custom_ops.py#L285